lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -39,6 +39,10 @@ use crate::vector_storage::{
     check_deleted_condition, new_stoppable_raw_scorer, VectorStorage, VectorStorageEnum,
 };
 
+// TODO(compressed_sparse_index): remove this once feature is released
+/// Whether to use the new compressed format.
+pub const USE_COMPRESSED: bool = false;
```

Comment:
Not strictly necessary, but maybe we can use this format. We use it in other places as well.

```suggestion
// TODO(1.10): remove this in the next minor release to expose the compressed sparse index
/// Whether to use the new compressed format.
pub const USE_COMPRESSED: bool = false;
```